src/main/kotlin/net/maizegenetics/phgv2/pathing/ImputationMetrics.kt

```diff
@@ -164,8 +175,10 @@ class ImputationMetrics : CliktCommand(help = "Impute best path(s) using read ma
                 val parentSampleRdCt = if (parentSampleHapid == "NA") 0 else hapidCounts.getOrDefault(parentSampleHapid, 0)
                 // check if the hapid and parentSampleHapid are the same
                 val hapidMatch = if (hapid != "NA" && parentSampleHapid != "NA" && hapid == parentSampleHapid) "true" else "false"
+                val refLen = hapidToSeqSize.getOrDefault(hapid, Pair(0,0)).second
+                val hapLen = hapidToSeqSize.getOrDefault(hapid, Pair(0,0)).first
 
-                writer.write("$refRange\t$hapid\t$sampleName\t$inImpuptation\t$inP39\t$parentSampleHapid\t$sampleRdCount\t$parentSampleRdCt\t$hapidMatch\n")
+                writer.write("$refRange\t$refLen\t$hapLen\t$hapid\t$sampleName\t$inImpuptation\t$inP39\t$parentSampleHapid\t$sampleRdCount\t$parentSampleRdCt\t$hapidMatch\n")
```

Comment:
Shouldn't inP39 be something like in target or something like that?

Reply:
I thought I had fixed that - let me see where it got messed up.

Reply:
Looks like I copied a wrong line - I"ll fix it.

Reply:
Acutally, this is correct.  If you look at the code above, the variable name is "inP39", which is just a true/false value.  I will name the variable something different so it isn't confusing.

Reply:
Note the actual column is defined as "In_${sampleHvcfName}", so in my test case it will show as "In_P39" (because P39 is my supposed parent), but this will vary based on user's sample name.